solr/bin/solr

```diff
@@ -2069,8 +2069,11 @@ if [ "$SOLR_MODE" == 'solrcloud' ]; then
     CLOUD_MODE_OPTS+=('-Dbootstrap_confdir=./solr/collection1/conf' '-Dcollection.configName=myconf' '-DnumShards=1')
   fi
 
+  if [ -n "$SOLR_SOLRXML_REQUIRED" ]; then
```

Comment:
Shouldn't this and the one be `== true`, like in the `.cmd` version?

Reply:
Fixed